packages/type-css-modules/.gitignore

```diff
@@ -1,5 +1,5 @@
 # compiled output
-/dist/
+# /dist/
```

Comment:
Wanted to see if the new `prepare` script could solve the compilation issue for `type-css-modules`, described in #73. Didn't seem like it.

```sh
[lint:types] > docs-app-for-embroider-css-modules-temporary@0.1.6 prelint:types
[lint:types] > type-css-modules --src app
[lint:types] 
[lint:types] sh: 1: type-css-modules: not found
[lint:types] npm run lint:types exited with code 127
```